numpy/core/fromnumeric.py

```diff
@@ -2516,6 +2520,17 @@ def cumsum(a, axis=None, dtype=None, out=None):
     array([[ 1,  3,  6],
            [ 4,  9, 15]])
 
+    ``cumsum(b)[-1]`` may not be equal to ``sum(b)``
+
+    >>> b = np.array([1, 2e-9, 3e-9] * 1000000)
+    >>> np.cumsum(b)
+    array([1.00000000e+00, 1.00000000e+00, 1.00000000e+00, ...,
+           1.00000001e+06, 1.00000001e+06, 1.00000001e+06])
```

Comment:
IMO this line doesn't add much to this particular example, so I'd just as soon remove it.

```suggestion
```